### PR TITLE
Images are functional now

### PR DESCRIPTION
**Pull request by sophtank**
Originally tried to implement the Imgur API but I wasn't able to finalize everything, so I switched gears.
Users are now asked to input a link to the image they would like to upload and the database should be able to handle everything else.
Currently there is no data validation as I wasn't sure how to approach it as all links will be different. Maybe regex on the string to find either www. or https:// but links are supported without either of those? Appreciate any thoughts.

### CODE REVIEW

**Review by Zzz4nder — COMMENTED** [review]
did you add this to the editdrive.html as well?

File: templates/createdrive.html [hunk 1]
```diff
@@ -71,7 +71,7 @@ <h2>Caption</h2>
     <input class="form-control" id="captionDrive" name="captionDrive" type="text" placeholder="Caption your drive" aria-label="Drive Caption" size="150px">
 <div class="mb-3">
     <h2>Add a photo from your drive</h2>
-    <input class="form-control" type="file" id="drivePhoto" name="drivePhoto">
+    <input class="form-control" id="drivePhoto" name="drivePhoto" type="text" placeholder="Please link to a photo of your drive">
```
Comment (Zzz4nder) [inline templates/createdrive.html:74]: Did you add this to the editDrive.html as well?

**Review by Zzz4nder — COMMENTED** [review]
I outlined the line of code I was referencing previously

File: templates/createdrive.html (hunk 1 above)
Comment (sophtank) [reply templates/createdrive.html:74]: I didn't, good catch. Appreciate it.

**Pull request merged**